packages/taraxa-sdk/src/DposClient.ts

```diff
@@ -0,0 +1,296 @@
+import { ethers } from 'ethers';
+import { DposAbi, DposAbi__factory as Dpos } from '../types/ethers-contracts';
+import {
+  DPOS_CONTRACT_ADDRESS,
+  Network,
+  NetworkName,
+  ProviderType,
+  getNetwork,
+  getNetworkById,
+} from './networks';
+
+export class DposClient {
+  private _dpos: DposAbi;

```

Comment:
```suggestion
  private dpos: DposAbi;

```
Naming with _ is a pattern in Js for private but since we're in Ts I don't think it is needed. However, it is up to your belief. I don't wanna nit on it

Reply:
I have a getter with that name that`s why I added the _